### DISCUSSION TITLE
Frequent full delete/full reinstall every few days?

### DISCUSSION THREAD

**Discussion by jtran-developer** [TheLastBen/fast-stable-diffusion#1410] [General]
Hi.  First of all, I'm very much appreciated in all the work that has gone into this.  I know it's a lot of hard work because of the constant updates.  

That's also why I'm writing in today.  Several times a week I'm forced to full delete/full reinstall the system because there was an update, my files weren't updated properly and something broke.  The latest error is the following:

AttributeError: 'Dropdown' object has no attribute 'multiselect'

This wasn't as much of a bother before because I could just let it run automatically, come back in 5 minutes, and it'll be done setting up by itself.  But now, there are so many extensions, settings and personal files interwoven within the sd folder (tagger model, lora, embs, etc) that doing a full delete/install is increasingly becoming more of a hassle.  I'm still going to use the program because it's amazing, but is there any way to remedy this?  Thank you.

**Comment by TheLastBen** [comment]
Hi, that issue is caused by the extensions, I have no control over them, you just have to choose which ones work without and issue.

**Reply by jtran-developer** [reply]
Thanks, looks like the current lora/additional network extension is breaking.  I did a fresh install and it broke as soon as I installed the extension and restarted the UI.  Is there any way to load up SD anyways despite one of its extensions breaking?  The extension should be non-critical right?  I just need SD to boot up in order to deal with any issues with the extensions.

**Reply by TheLastBen** [reply]
copy the full error log so we can find a way to boot sd

**Reply by deceani** [reply]
I got the same error just now after update

`Traceback (most recent call last):
  File "D:\StableDiffusion\stable-diffusion-webui\launch.py", line 317, in <module>
    start()
  File "D:\StableDiffusion\stable-diffusion-webui\launch.py", line 312, in start
    webui.webui()
  File "D:\StableDiffusion\stable-diffusion-webui\webui.py", line 159, in webui
    shared.demo = modules.ui.create_ui()
  File "D:\StableDiffusion\stable-diffusion-webui\modules\ui.py", line 1807, in create_ui
    visit(txt2img_interface, loadsave, "txt2img")
  File "D:\StableDiffusion\stable-diffusion-webui\modules\ui.py", line 168, in visit
    visit(c, func, path)
  File "D:\StableDiffusion\stable-diffusion-webui\modules\ui.py", line 168, in visit
    visit(c, func, path)
  File "D:\StableDiffusion\stable-diffusion-webui\modules\ui.py", line 168, in visit
    visit(c, func, path)
  [Previous line repeated 2 more times]
  File "D:\StableDiffusion\stable-diffusion-webui\modules\ui.py", line 170, in visit
    func(path + "/" + str(x.label), x)
  File "D:\StableDiffusion\stable-diffusion-webui\modules\ui.py", line 1805, in loadsave
    apply_field(x, 'value', check_dropdown, getattr(x, 'init_field', None))
  File "D:\StableDiffusion\stable-diffusion-webui\modules\ui.py", line 1770, in apply_field
    elif condition and not condition(saved_value):
  File "D:\StableDiffusion\stable-diffusion-webui\modules\ui.py", line 1800, in check_dropdown
    if x.multiselect:
AttributeError: 'Dropdown' object has no attribute 'multiselect'`

**Reply by jtran-developer** [reply]
Yeah, that's the error message I got.  I went into the ui.py, removed that one method and it booted up properly.  It would be nice to be able to boot up SD if nothing critical broke, and not having to wait for someone to come save us though.

**Comment by TheLastBen** [comment]
are you using launch.py ?

**Reply by jtran-developer** [reply]
I can guess what that does, but I don't know how to use it as a py file.  I just press the play button.

**Reply by TheLastBen** [reply]
if you encounter too many issues, you can simply skip the first cell "gdrive" and work with the temporary storage to avoid extensions conflict

**Comment by TheLastBen** [comment]
I just found out that the issue is caused by A1111 upgrading to gradio 3.16.2

**Reply by jtran-developer** [reply]
Thanks, as usual it was quickly resolved.  However, a new error appeared a few days ago.  Luckily, it was something I could just ignore and SD would still boot up.  However however, yet another error appeared just now, which breaks the program entirely.  This time, even a delete everything/fresh install won't resolve the issue.  Is it possible to disable updates entirely and i'll just update it manually every other week or so?  Here's today's error:
```
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 239, in <module>
    webui()
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 169, in webui
    shared.demo = modules.ui.create_ui();shared.demo.queue(concurrency_count=999999,status_update_rate=0.1)
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/ui.py", line 947, in create_ui
    fn=wrap_gradio_call(modules.extras.run_pnginfo),
AttributeError: module 'modules' has no attribute 'extras'
```

I went ahead and updated the code on my end to import it, but then another error popped up

**Reply by TheLastBen** [reply]
extras is a py file inside the folder "modules", check that it exists in your gdrive

**Reply by jtran-developer** [reply]
It's working now.  I tried a full delete/fresh install twice earlier and it didn't work.  I went into the ui.py file and the import module.extras line was missing both times.  I waited awhile, did a 3rd full delete/fresh install and now it works.  I checked the file again and the import line is there now.
